### PR TITLE
Correct check for activity is root in OTLP

### LINKED ISSUE #1288: Traces are rejected by SignalFx because Root Span contains ParentSpanId=00000000
# Bug Report

List of [NuGet packages](https://www.nuget.org/profiles/OpenTelemetry) and
version that you are using (e.g. `OpenTelemetry 0.4.0-beta.2`):

* OpenTelemetry -Version 0.6.0-beta.7

Runtime version (e.g. `net461`, `net48`, `netcoreapp2.1`, `netcoreapp3.1`, etc.
You can find this information from the `*.csproj` file):

* netcoreapp3.1

## Symptom

We were trying to export traces to SignalFx but they were rejected because the root span contained a property ParentSpanId equal to' 00000000' instead of 'null'.  

We noticed this test was commented out which would identify the issue.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/test/OpenTelemetry.Tests/Trace/TracerProviderSdkTest.cs#L47

This is the Span definition in the proto files https://github.com/open-telemetry/opentelemetry-proto/blob/master/opentelemetry/proto/trace/v1/trace.proto#L82

## Reproduce

I used the example code from here and added the otlpExporter: https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/examples/Console/TestConsoleExporter.cs#L43-L47

We will close this issue if:

* The repro project you share with us is complex. We can't investigate custom
  projects, so don't point us to such, please.
* If we can not reproduce the behavior you're reporting.

### PR DESCRIPTION
**Pull request by cijothomas**
Workaround and Fixes #1288.

Won't need workaround once this is fixed in .NET runtime itself :https://github.com/dotnet/runtime/issues/42456

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs [hunk 1]
```diff
@@ -87,7 +89,7 @@ internal static OtlpTrace.Span ToOtlpSpan(this Activity activity)
             activity.SpanId.CopyTo(spanIdBytes);
 
             var parentSpanIdString = ByteString.Empty;
-            if (activity.ParentSpanId != default)
+            if (activity.ParentSpanId.ToHexString() != EmptyActivitySpanId)
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs:92]: Probably put a TODO comment here so we can revisit after .NET fixed this?

**Review by reyang — APPROVED** [review]
LGTM.

**Review by CodeBlanch — APPROVED** [review]
LGTM

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs [hunk 2]
```diff
@@ -91,6 +91,10 @@ internal static OtlpTrace.Span ToOtlpSpan(this Activity activity)
             var parentSpanIdString = ByteString.Empty;
             if (activity.ParentSpanId.ToHexString() != EmptyActivitySpanId)
             {
+                // Once .NET fixes this https://github.com/dotnet/runtime/issues/42456
```
Comment (eddynaka) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs:94]: If you can, start with TODO

Comment (eddynaka) [reply src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs]: just updated.

**Pull request merged**